pypeit/core/wavecal/autoid.py

```diff
@@ -1324,7 +1344,15 @@ def echelle_wvcalib(spec, orders, spec_arxiv, wave_arxiv, lamps, par,
             continue
         # ToDO should we still be populating wave_calib with an empty dict here?
         if iord not in ok_mask:
+            msgs.warn(f"Skipping order = {orders[iord]} ({iord+1}/{norders}) because masked")
+            wv_calib[str(iord)] = None
+            all_patt_dict[str(iord)] = None
+            continue
+        if np.all(spec_arxiv[:, iord] == 0.0):
```

Comment:
This may be redundant at this point, because a similar condition is present in `predict_ech_arcspec()` in `pypeit/core/wavecal/echelle.py`.  
It may happen when the wavelength solution guess is outside the range of the observations, and therefore the values are all zeros at the range of the observations.
But before, with an all zeros arxiv, reidentify was still trying to cross-correlate with the observed spectrum, while this condition here avoid that.